internal/olm/operator/manager.go

```diff
@@ -223,7 +229,7 @@ func (m *operatorManager) run(ctx context.Context) (err error) {
 
 func (m *operatorManager) cleanup(ctx context.Context) (err error) {
```

Comment:
```suggestion
// cleanup will remove the resources which are applied in the OLM namespace to test the operator integrated with,
func (m *operatorManager) cleanup(ctx context.Context) (err error) {
```

Reply:
Could you please add the comment in this one? Then, it is helpful if we understand the code.